docs/manual-gce-deployment.md

```diff
@@ -0,0 +1,374 @@
+
+That script can be found [here](https://github.com/copejon/gk-cluster-deploy/blob/master/deploy/vm/do-startup.sh).
+
+## Step by Step Deployment
+
+*NOTE: All actions are meant to be executed as **root** user.*
+
+### Laying the Foundation
+
+#### On All Nodes...
+
+1. Stop and Disable `firewalld`
+
+    `# systemctl stop firewalld && systemctl disable firewalld`
+
+2. Create the kubernetes yum repo
+
+    ```
+    su -c "cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+    [kubernetes]
+    name=Kubernetes
+    baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+    enabled=1
+    gpgcheck=1
+    repo_gpgcheck=1
+    gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+    https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+    EOF"
+    ```
+
+3. Yum Install RPMs
+
+    `# yum install -y docker-1.12.6 glusterfs-fuse kubelet kubeadm`
+
+4. Enable and Start Docker
+
+    `# systemctl enable docker && systemctl start docker`
+
+4. Install `kubectl`
+
+  - Get the binary
+
+    `# curl -sSLO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl`
+
+  - Make the binary executable
+
+    `# chmod +x ./kubectl`
+
+  - Move the binary to `bin` dir
+
+    `# mv ./kubectl /usr/bin/`
+
+
+#### On the Master Node
+
+1. Install Master RPMs
+
+    `# yum install -y git socat`
+
+2. Clone [CNS Object Broker](https://github.com/copejon/cns-object-broker)
+
+    `# git clone https://github.com/copejon/cns-object-broker.git`
+
+3. Clone [Gluster-Kubernetes](https://github.com/gluster/gluster-kubernetes)
+
+    `# git clone https://github.com/gluster/gluster-kubernetes.git`
+
+4. Install `helm` binary
+
+  - Get the binary
+
+    `# curl -sSL https://storage.googleapis.com/kubernetes-helm/helm-v2.5.0-linux-amd64.tar.gz | tar zx`
+
+  - Move the binary to `bin` dir
+
+    `# mv $(find /tmp/ -name helm) /usr/bin/`
+
+5. Install `heketi-cli`
+
+  - Get the binary
+
+    `# curl -sSL https://github.com/heketi/heketi/releases/download/v4.0.0/heketi-client-v4.0.0.linux.amd64.tar.gz | tar -xz`
+
+  - Move the binary to `bin` dir
+
+    `# mv $(find ./ -name heketi-cli) /usr/bin/`
+
+6. Initialize the kubernetes master node.
+
+    The following command initializes the master node *and* Flannel as the pod network.
+    If you would prefer to use a different pod network, see the [kubernetes documentation](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network) for other options.
+
+    `# kubeadm init --pod-network-cidr=10.244.0.0/16`
+
+    **Note the `kubeadm join` command output be `kubeadm init`!  This is used to connect minions to the master.**
+
+#### Initialize and Connect Minions to the Master
+
+**NOTE: This requires the `kubeadm join` command output by `kubeadm init`.**
```

Comment:
Is this note a dup of the note 1 line above?

Reply:
Not a dupe, but maybe redundant.  Wanted to stress that the command mentioned in the above Note is required in this section